### PR TITLE
HW1_Zhenwei_Dai

### PR DESCRIPTION
**Pull request by ChovyDavid**
pr

### CODE REVIEW

**Comment by fzybb** [comment]
Detailed explanation in hw1, well done.

**Comment by iTvX** [comment]
very detailed, good job

File: ShortQuestions/HW1.md
Comment (QueenieOhYeah) [inline ShortQuestions/HW1.md:1]: Your answers are detailed. Good job!

**Comment by yuants4213** [comment]
The information is thorough, and the arrangement is easily understandable.

**Comment by Liam-Zhou** [comment]
resolve the conflicts

**Pull request merged**